### PR TITLE
feat: add filter to TreeDetailsPage

### PR DESCRIPTION
**Pull request by lfjnascimento**
add filter to TreeDetailsPage

## How to test it

- use docker compose to run the services and go to a TreeDetails page, then checkout the FilterDrawer

### CODE REVIEW

File: dashboard/src/locales/messages/index.ts [hunk 1]
```diff
@@ -5,6 +5,11 @@ export const messages = {
     global: {
       filters: 'Filters',
       cleanAll: 'Clean all',
+      branch: 'Branch',
+      status: 'Status',
+      configs: 'Configs',
+      architecture: 'Architecture',
+      timing: 'Timing',
```
Comment (mari1912) [inline dashboard/src/locales/messages/index.ts:12]: nit: whenever is possible I think it is better to follow alphabetical order

File: dashboard/src/locales/messages/index.ts [hunk 2]
```diff
@@ -39,6 +44,11 @@ export const messages = {
       filtering: 'Filtering',
       treeURL: 'Tree URL',
       refresh: 'Refresh',
+      branchSubtitle: 'Please select one or more Branches:',
+      statusSubtitle: 'Please select one or more Status:',
+      configsSubtitle: 'Please select one or more configs:',
+      architectureSubtitle: 'Please select one or more Architectures:',
+      timingSubtitle: 'Please select a range of timing:',
```
Comment (mari1912) [inline dashboard/src/locales/messages/index.ts:51]: same

File: dashboard/src/routes/TreeDetails/TreeDetailsFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+};
+
+export default TreeDetailsFilter;
+
+const summarySectionProps = {
+  title: 'Tree',
+  columns: [
+    { title: 'Tree', value: 'stable-rc' },
+    { title: 'Matainer', value: '' },
+    { title: 'Estimate to complete', value: '' },
```
Comment (mari1912) [inline dashboard/src/routes/TreeDetails/TreeDetailsFilter.tsx:126]: this values should be using i18n

**Review by mari1912 — APPROVED** [review]

File: dashboard/src/routes/TreeDetails/TreeDetailsFilter.tsx (hunk 3 above)
Comment (lfjnascimento) [reply dashboard/src/routes/TreeDetails/TreeDetailsFilter.tsx:126]: actually this should come from the backend, I will implement this soon

**Pull request merged**